CoreTimelineView/src/au/gov/asd/tac/constellation/views/timeline/TimelineTopComponent.java

```diff
@@ -170,7 +167,7 @@ public TimelineTopComponent() {
 
             // Create the scene:
             final Scene scene = new Scene(root);
-            scene.getStylesheets().add(JavafxStyleManager.getMainStyleSheet());
+            scene.getStylesheets().addAll(JavafxStyleManager.getMainStyleSheet());
```

Comment:
I've seen this in a few places, but what is reason for switching from add to addAll given it seems only one file is being added?

Reply:
![image](https://github.com/constellation-app/constellation/assets/139201041/589c82ce-a2be-4d06-95a0-a0d6b9ae74ea)